File: Services/VM/ProfileData.cs
using Model;
using System;
using System.Collections.Generic;
using System.Text;

namespace Services.VM
{
    public class ProfileData
    {
        public Model.Users User { get; set; }
        public List<Model.JobOffer> TakenbyUser { get; set; }
        public List<Model.JobOffer> AddandTaken { get; set; }
        public List<Model.JobOffer> AddedByUser { get; set; }
    }
}

File: JobHunter/ClientApp/src/app/models/profileData.ts
import { TakenOffer, JobOffer } from "./jobOffer";
import { Users } from "./users";

export class ProfileData {
  public takenByUser: TakenOffer[];
  public addedByUser: JobOffer[];
  public addandTaken: TakenOffer[];
  public user: Users;
  public editingProfile: boolean;
}
